### PR TITLE
Fix toast on success on downvote

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- toast on success on downvote
- fix checking for nr of comments in comments button rather then upvotes
- share menu positioning fix

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices? No
- [ ] iOS (Chrome and Safari)
- [ ] Android

### CODE REVIEW

File: packages/shared/src/components/cards/v1/ActionButtons.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default function ActionButtons({
               id={`post-${post.id}-comment-btn`}
               className={classNames(
                 'pointer-events-auto ml-2',
-                post?.numUpvotes > 0 ? '!pl-3' : '!px-1',
+                post?.numComments > 0 ? '!pl-3' : '!px-1',
```
Comment (kopancek) [inline packages/shared/src/components/cards/v1/ActionButtons.tsx:157]: nice, thanks!

**Review by kopancek — APPROVED** [review]
LGTM :shipit:

File: packages/shared/src/components/cards/v1/ActionButtons.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ export default function ActionButtons({
           <PostTagsPanel
             post={post}
             className="pointer-events-auto mt-4"
-            toastOnSuccess={false}
+            toastOnSuccess
```
Comment (rebelchris) [inline packages/shared/src/components/cards/v1/ActionButtons.tsx:91]: Default is true already, so we can omit this.
```suggestion
           
```

**Review by rebelchris — APPROVED** [review]
Minor query, but not blocking

**Pull request merged**